site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -620,7 +646,82 @@ components:
          description: Allows arbitrary properties as extension to the ones
                       specified by the Hono API.
 
-# Tenant schema
+
+      PagingSortingFilteringOptions:
+         type: object
+         additionalProperties: false
+         properties:
+            "page":
+               $ref: '#/components/schemas/PagingOptions'
+            "filters":
+               $ref: '#/components/schemas/FilterOptionsList'
```

Comment:
how about using
```yaml
"filters":
  type: array
  items:
    $ref: '#/components/schemas/FilterOption'
```

in order to not having to create a separate object definition for each list of objects?